pkgs/applications/networking/p2p/stig/default.nix

```diff
@@ -38,15 +38,22 @@ python3.pkgs.buildPythonApplication rec {
     setproctitle
   ];
 
-  checkInputs = with python3.pkgs; [
+  checkInputs = with python3Packages; [
     asynctest
-    pytest
+    pytestCheckHook
   ];
 
-  checkPhase = ''
-    pytest tests
+  dontUseSetuptoolsCheck = true;
+
+  preCheck = ''
+    export LC_ALL=C
+    export HOME=$(mktemp -d)
```

Comment:
```suggestion
    export HOME=$TMPDIR
```

Reply:
The manual suggests `HOME=$(mkdir -d)`, but both can be found in nixpkgs.
However I introduced it before setting `dontUseSetuptoolsCheck = true;` when I hit the following error:
```
Can't create directory /homeless-shelter/.local/share/stig/histories: Permission denied
```
But with `dontUseSetuptoolsCheck = true;` it doesn't fail, so let's remove it I guess.